bika/lims/browser/worksheet/templates/add_analyses.pt

```diff
@@ -43,14 +43,15 @@
 				<input type="hidden" name="submitted" value="1"/>
 				<input tal:replace="structure context/@@authenticator/authenticator"/>
 				<label for="getWorksheetTemplate" i18n:translate="">Select template</label>
-				<select id="ProfileSelector" name="getWorksheetTemplate">
+				<select id="ProfileSelector" name="getWorksheetTemplate"
+					tal:define="olist view/getWorksheetTemplates">
```

Comment:
Better to use a var name more meaningful, like `ws_templates_list` instead of `olist`